util/conversion/conversion.go

```diff
@@ -30,13 +35,50 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/serializer"
 	"k8s.io/apimachinery/pkg/util/json"
 	"k8s.io/utils/diff"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/conversion"
 )
 
 const (
 	DataAnnotation = "cluster.x-k8s.io/conversion-data"
 )
 
+var (
+	contract = clusterv1.GroupVersion.String()
```

Comment:
Is this going to be error prone having to remember to update the import path above? I'm wondering if it would make sense to have this be a field that is passed in, which would also allow downstream users to consume it for their own potential use as well.

Reply:
Either that or a utility defined in the /api/<version> package?

Reply:
Last time we rewrote all the import paths, I think that would work well for here